vividus-extension-selenium/src/main/java/org/vividus/ui/action/ExpectedSearchActionsConditions.java

```diff
@@ -32,7 +32,6 @@ public final class ExpectedSearchActionsConditions extends AbstractExpectedCondi
     @Override
     protected List<WebElement> findElements(SearchContext searchContext, Locator locator)
     {
-        locator.getSearchParameters().setWaitForElement(false);
```

Comment:
I don't think that's correct to remove it from here.
The usage of the waiter here involves the application of presenceAllElements condition.
We currently have this usage of the same invisibility condition
<img width="451" alt="image" src="https://user-images.githubusercontent.com/17816729/183844399-f2cb0b6c-e963-4437-aabb-2450baf38375.png">

Reply:
This is neither correct place to have this invocation since this toooo generic to manipulate locator parameters, its your concern that for each invocation of presenceOfAllElementsLocatedBy in steps we should mutate input locator?

Reply:
1. If you feel lucky you could change search actions api to allow search without default waiting
2. In this implementation, every single condition receives presenceOfAllElementsLocatedBy by default. It is not transparent nor correct.